### PR TITLE
Fix Axis Praxis movie in Variable Fonts article

### PR DESCRIPTION
**Pull request by RoelN**
This MP4 movie has been reencoded to match the other videos on the
page, as this movie otherwise wouldn't play in Firefox:

afbeelding

This MP4 file should be moved to your cloud storage bucket, as done when we wrote this article: https://github.com/GoogleChrome/web.dev/pull/3445#issuecomment-692241527 @kaycebasques Is that something I can bother you for? :-)

### CODE REVIEW

**Comment by RoelN** [comment]
Note that the staging will not show the fixed movie, as it will need to be moved to the cloud storage bucket first.

**Comment by mihajlija** [comment]
@RoelN Thank you for the fix, adding this to my TODO list and I'll make sure the video is uploaded by Monday.

**Comment by davelab6** [comment]
Thank you @mihajlija!

File: src/site/content/en/blog/variable-fonts/index.md [hunk 1]
```diff
@@ -643,9 +643,9 @@ You can view the working example and source code for the above sample
 ### Animation
 
 <figure class="w-figure">
-  <video controls autoplay loop muted class="w-screenshot">
-    <source src="https://storage.googleapis.com/web-dev-assets/variable-fonts/axis-praxis.mp4" type="video/mp4">
-  </video>
+  {% Video src="video/vgdbNJBYHma2o62ZqYmcnkq3j0o1/2Du2L0Ii5nUqz8n6S3Vz.mp4",
+  class="w-screenshot"
```
Comment (mihajlija) [inline src/site/content/en/blog/variable-fonts/index.md:647]:
```suggestion
  {% Video src="video/vgdbNJBYHma2o62ZqYmcnkq3j0o1/2Du2L0Ii5nUqz8n6S3Vz.mp4",
  class="w-screenshot",
  controls=false,
  autoplay=true,
  loop=true,
  muted=true
```

**Review by mihajlija — APPROVED** [review]

**Comment by RoelN** [comment]
@mihajlija Thanks picking this up!

Re: https://github.com/GoogleChrome/web.dev/pull/4746/commits/ee6b781b044e9db927a38cccedea768586b6b868, I think the `{% video %}` component sets these automatically so they're not set for other videos in this document. So they might not be needed? Unless you explicitly want to turn off controls, which the other videos still have. Not sure what's here but figured I'd point this out for consistency's sake.

**Comment by mihajlija** [comment]
The `{% Video %}` component is a new feature on web.dev, it creates a video that doesn't loop, or autoplay, with the controls visible by default, so the attributes are necessary to make this video behave like all the others on the page. From what I can tell, the controls are visible on hover on all the videos.

**Comment by RoelN** [comment]
@mihajlija Gotcha now, thanks for the explanation. Thanks for picking this up!

**Pull request merged**